qemu/softmmu_template.h

```diff
@@ -178,23 +178,31 @@ WORD_TYPE helper_le_ld_name(CPUArchState *env, target_ulong addr, int mmu_idx,
     uintptr_t haddr;
     DATA_TYPE res;
     int error_code;
+    struct hook *hook;
+    int handled;
```

Comment:
"handled" should have type "bool", not "int", because "uc_cb_eventmem_t" return "bool" value